app/helpers/datasets_helper.rb

```diff
@@ -23,4 +36,26 @@ def documentation?(key)
     key != nil && key != ""
   end
 
+  def expected_update
+    last = Time.parse(@dataset["updated_at"])
```

Comment:
I think if we use updated_at a change in say, summary, will result in this being wrong.  I think it should really use the end_date of the most recent 'link' to be accurate.